<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LampWidget</class>
 <widget class="QGroupBox" name="LampWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>552</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>GroupBox</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,8">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="6,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Brillo:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sliderBright">
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>150</height>
          </size>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="pageStep">
          <number>20</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>25</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QToolButton" name="btnOff">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>1</width>
         <height>1</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>15</width>
         <height>15</height>
        </size>
       </property>
       <property name="text">
        <string>Off</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="colorSelectorLy">
     <property name="spacing">
      <number>0</number>
     </property>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
